lib/streamlit/elements/exception_proto.py

```diff
@@ -142,6 +194,18 @@ def get_stack_trace(exception, exception_traceback=None):
             "Try calling exception() within the `catch` block."
         ]
     else:
+        if strip_streamlit_stack_entries:
+
+            def get_stackframe_filename(frame):
+                # Python 3 has a frame.filename variable, but frames in
+                # Python 2 are just tuples. This code works in both versions.
+                return frame[0]
```

Comment:
Why is this function defined at this level? Move to top level to make it clear that there is no closure.